### ISSUE TITLE
Error in rifle.conf -- Copied and pasted your example version verbatim

### ISSUE THREAD

**Issue by dsifford** [hut/ranger#474]
Hello,

I keep getting this error when I exit ranger

```sh
Syntax error in /home/dsifford/.config/ranger/rifle.conf line 73 (Line without delimiter)
```

I got the above error using this file exactly as is: https://github.com/hut/ranger/blob/master/ranger/config/rifle.conf

Any tips would be greatly appreciated. Thank you!

**Comment by hut** [comment]
I noticed that this error message is triggered at lines that contain only whitespace, which should instead be silently ignored. I pushed some commits, can you pull them and try again?

**Comment by dsifford** [comment]
Sure can.. I'll do that now and report back. Thanks! :beers:

**Comment by dsifford** [comment]
@hut Success! Thanks so much

Of note: While installing the latest version from this repo, I noticed that the one I got from the apt ppa is version 1.6.1! That's like 3 years old! 

Do you have a ppa somewhere I can add to my sources list?

Thanks!

**Comment by hut** [comment]
Yay!

Btw, you could've also fixed the issue by deleting your rifle.conf, because it is not needed unless you want to actually change it yourself.

I don't do packaging for ranger, I guess your best bet is to simply install it manually, since the versions don't change so often anyway:

```sh
git checkout stable
sudo make install
```

**Comment by dsifford** [comment]
@hut The `rifle.conf` that I use normally is heavily modified. Just wanted to be absolutely sure that it wasn't user-error (which is usually the case) :grin:

Re: Packaging -- Noted, will do! ... One day perhaps I'll bite the bullet and switch to an arch distro... Ubuntu is sooo behind on absolutely everything! 


---

Also, while I've got your attention, I have an unrelated question that I've been trying to figure out recently but have been so far unsucessful..

Is there a way that I can use ranger as a quick and dirty `cd` command? As an example of what I'm describing, consider the following:

Assume my starting directory = `~`

1. `ranger` 
2. \*\*use ranger to move to some deeply nested directory somewhere else\*\*
3. `q` back to the terminal at the directory I just navigated to

Possible? Thanks for all the help!
